BoyumFoosballStats-2.0/Pages/_Host.cshtml

```diff
@@ -16,7 +16,7 @@
     <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" rel="stylesheet"/>
     <link href="_content/MudBlazor/MudBlazor.min.css" rel="stylesheet"/>
     <link href="css/site.css" rel="stylesheet"/>
-    <link href="BoyumFoosballStats_2.styles.css" rel="stylesheet"/>
+    @* <link href="BoyumFoosballStats_2.styles.css" rel="stylesheet"/> *@
```

Comment:
Should this be removed entirely?

Reply:
Oh, I had a weird styling issue and was trying to figure where it comes from. This is responsible for scoped css, I can see that we might use it. Idk...